tests/code_update.py

```diff
@@ -176,8 +149,8 @@ def run(args):
         network.start_and_join(args)
 
         test_verify_quotes(network, args)
+        test_add_node_with_bad_code(network, args)
         test_update_all_nodes(network, args)
-        test_verify_quotes(network, args)
```

Comment:
I thought having this test at the end was useful, to make sure we can verify quotes after all nodes have been updated.

Reply:
It's not clear to me what this is testing that the first one isn't: nodes are already added and we check their quotes. This is doing it on a different set of nodes, but the procedure to add them is identical and checks for quote exactly the same way.

I think this was meaningful in the fat genesis day, when we did not check the quotes of the original set of nodes at all. Now, not so much. Anyway, I've put it back, in case I've missed a reason why it's useful :)